tasks/task086_translated_symbol_arithmetic.json

```diff
@@ -9,24 +9,24 @@
         "Answer Generation",
         "Arithmetic"
     ],
-    "Definition": "The operattions 'addition' and 'subtraction' have been replaced with their italian translations i.e you need to perform addition when you see 'aggiunta' and subtraction  in case of 'sottrazione'.",
+    "Definition": "In this task You will be given an arithmetic operation and you have to find its answer. The operations 'addition' and 'subtraction' have been replaced with their italian translations i.e you need to perform addition when you see 'aggiunta' and subtraction in case of 'sottrazione'.",
     "Positive Examples": [
         {
             "input": "10 sottrazione 7",
             "output": "3",
-            "explanation": "Here, 'sottrazione' represents subtraction operation. So, the answer is 3 (10-7=3)."
+            "explanation": "Here, 'sottrazione' represents the subtraction operation. So, the answer is 3 (10-7=3)."
         },
         {
             "input": "11 aggiunta 20 sottrazione 3",
             "output": "28",
-            "explanation": "Here, 'aggiunta' represents addition operation and 'sottrazione' represents subtraction operation. So, the answer is 28 (11+20-3=28)."
+            "explanation": "Here, 'aggiunta' represents the addition operation and 'sottrazione' represents the subtraction operation. So, the answer is 28 (11+20-3=28)."
         }
     ],
     "Negative Examples": [
         {
             "input": "1 aggiunta 8 sottrazione 100",
-            "output": "20",
-            "explanation": "Here, 'aggiunta' represents addition operation and 'sottrazione' represents subtraction operation. So, the answer should have been -91 (1+8-100=-91) instead of 20."
+            "output": "93",
+            "explanation": "Be careful that 'aggiunta' represents the addition operation and 'sottrazione' represents the subtraction operation. Using these operations instead of one-another can lead to incorrect answers."
```

Comment:
I found the previous explanation of "So, the answer should have been -91 (1+8-100=-91) instead of 20." helpful in explaining why this is an incorrect answer. I suggest we bring it back.